src/asm/main.c

```diff
@@ -6,815 +6,97 @@
 /*   By: asarandi <asarandi@student.42.us.org>      +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2018/02/01 04:11:28 by asarandi          #+#    #+#             */
-/*   Updated: 2018/02/11 06:43:16 by asarandi         ###   ########.fr       */
+/*   Updated: 2018/03/02 02:39:28 by asarandi         ###   ########.fr       */
```

Comment:
It seems this entire file was normed?  The diff looks awful which is why I asked.

Reply:
Everything was written in the same file at first, then got split out into individual files.